content/advanced/430_emr_on_eks/prereqs.md

```diff
@@ -113,3 +113,44 @@ After you register, you should get confirmation that your EMR virtual cluster is
     "name": "eksworkshop-eksctl",
     "arn": "arn:aws:emr-containers:us-west-2:xxxxxxxxxxxx:/virtualclusters/av6h2hk8fsyu12m5ru8zjg8ht"
 {{< /output >}}
+
+### Create EKS Managed Node Group
+
+Lets add a EKS managed nodegroup to this EKS cluster to have more resources to run sample spark jobs.
+
+Create a config file (addnodegroup.yaml) with details of a new EKS managed nodegroup. 
+
+```sh
+cat << EOF > addnodegroup.yaml
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: eksworkshop-eksctl
+  region: ${AWS_REGION}
+
+managedNodeGroups:
+- name: emrnodegroup
+  desiredCapacity: 3
+  instanceType: m5.large
```

Comment:
Let's change the instance type to m5.xlarge to accommodate jobs with 2 vcpus executor size in later chapters.

Reply:
Changed the instance size to m5.xlarge and modified spark.executor instances in autoscaling chapter to show CA.